impexp-config/src/main/java/org/citydb/config/geometry/GeometryObject.java

```diff
@@ -291,6 +291,10 @@ public void changeSrid(int srid) {
     }
 
     public GeometryObject toEnvelope() {
+        if (geometryType == GeometryType.ENVELOPE) {
+            return this;
```

Comment:
`return this` may lead to an inconsistent behaviour of this function, because a new envelope object is expected to be returned.

Reply:
You are right, will adapt this.

Reply:
see here 474aa1acb54f8a6850eb9b46ae5ca1d87a6f0a01